geopandas/tests/test_crs.py

```diff
@@ -293,6 +295,30 @@ def test_dataframe(self):
         assert df.geometry.crs == self.wgs
         assert df.geometry.values.crs == self.wgs
 
+        # new geometry with set CRS has priority over GDF CRS -
```

Comment:
This test is already quite long, so can you maybe add it as a new test? (eg `test_dataframe_setitem`)

Reply:
Thanks for the feedback, they're both good ideas, I'll get onto that.